spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/fixtures/MultipleMappingFixture.scala

```diff
@@ -88,9 +88,12 @@ trait MultipleMappingFixture extends BeforeAndAfterAll with SparkTestBase {
       .mode(SaveMode.Overwrite)
       .parquet(getRawDataPath(mappingDS.hdfsPath))
     val value = "../examples/data/e2e_tests/data/hdfs/std/mappingConformanceRuleData/2020/03/23/v1/_INFO"
-    val infoFileContents: String = fromFile(value, "UTF-8").mkString
+    val fileContent = fromFile(value, "UTF-8")
```

Comment:
Just a few line below (originally one), it has been suggested to not use the `"UTF-8"` literal string but instead `Charset.defaultCharset`. But please do use the `FileReader` and if the encoding is needed, enhance the function there.